README.md

```diff
@@ -1,2 +1,40 @@
 # mAWS.jl
 Mocking Framework for AWS.jl
+
+## Overview
+This package generates patch functions for all AWS services which define them in the [aws-sdk-js](https://github.com/aws/aws-sdk-js/tree/master/apis) repository.
```

Comment:
Maybe add/link instructions on how to generate the code in `src/patches`, under a `## Contributing` section?

Reply:
The `.github/workflows/update_patches.yml` runs every night to regenerate and create a merge request if patches are added.

But will this in regardless!